bcda/testUtils/utils.go

```diff
@@ -140,3 +141,14 @@ func RandomBase64(n int) string {
 	}
 	return base64.StdEncoding.EncodeToString(b)
 }
+
+func GetEnvInt(varName string, defaultVal int) int {
```

Comment:
Is this meant to be here _and_ in utils/common.go?

Reply:
no. I moved it to testutils at first by mistake.  It should just be in utils now.

Reply:
> Is this ready to be reviewed? The description looks like it might be in progress still...?

This sb gtg now.